src/common/style/icon_color_css.ts

```diff
@@ -1,5 +1,23 @@
 import { css } from "lit-element";
 
+export interface StateColor {
```

Comment:
Have you thought about how this will work if someone wants to add number ranges

Reply:
Haha, I did but didn't want to do too much in this PR. I would just use the common conditional validation logic, which in someways would be a bit cleaner for maintenance.

Reply:
I would vote against that :P 

To keep it simpler, I would just add a single `below: number` key. Convert the state to a number and the first object that has `below > state`, you return that color.

Reply:
Sure. Simpler the better.